seq/src/main.rs

```diff
@@ -0,0 +1,125 @@
+use std::process;
+
+use clap::{load_yaml, App, AppSettings};
+
+#[derive(Clone,Debug)]
+pub struct Seq {
+    current: f64,
+    stop: f64,
+    step: f64,
+}
+
+impl Iterator for Seq {
+    type Item = f64;
+
+    fn next(&mut self) -> Option<Self::Item> {
+        if self.step.is_sign_positive() && self.current <= self.stop {
+            let result = self.current.clone();
```

Comment:
`f64` is `Copy`, so you don't need to clone

Reply:
Understood